pkg/builder/intg_test_context.go

```diff
@@ -0,0 +1,273 @@
+// Copyright (c) 2019 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package builder
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+
+	"github.com/go-logr/logr"
+	"github.com/google/uuid"
```

Comment:
nit: re-arrange the imports and include all the external imports to a single list